### PR TITLE
Improve shortlist formatting

### PR DESCRIPTION
**Pull request by hjmjohnson**
Make line breaks to more easily read the short listing of command line options
in one condensed setting. The previous line breaking produced a format that
was hard to parse, read, and copy for understanding command line options.

OLD (21 lines):
```
   /Users/johnsonhj/src/BT-11/bin/BRAINSLandmarkInitializer
                                        [--returnparameterfile
                                        <std::string>]
                                        [--processinformationaddress
                                        <std::string>] [--xml] [--echo]
                                        [--bsplineNumberOfControlPoints
                                        <int>]
                                        [--inputReferenceImageFilename
                                        <std::string>]
                                        [--outputTransformType
                                        <AffineTransform|BSplineTransform
                                        |VersorRigid3DTransform>]
                                        [--outputTransformFilename
                                        <std::string>]
                                        [--inputWeightFilename
                                        <std::string>]
                                        [--inputMovingLandmarkFilename
                                        <std::string>]
                                        [--inputFixedLandmarkFilename
                                        <std::string>] [--] [--version]
                                        [-h]

```

NEW (16 lines):
```
    /Users/johnsonhj/src/BT-11/bin/BRAINSLandmarkInitializer
     [--returnparameterfile <std::string>]
     [--processinformationaddress <std::string>]
     [--xml]
     [--echo]
     [--bsplineNumberOfControlPoints <int>]
     [--inputReferenceImageFilename <std::string>]
     [--outputTransformType <AffineTransform|BSplineTransform
                |VersorRigid3DTransform>]
     [--outputTransformFilename <std::string>]
     [--inputWeightFilename <std::string>]
     [--inputMovingLandmarkFilename <std::string>]
     [--inputFixedLandmarkFilename <std::string>]
     [--]
     [--version]
     [-h]
```

### CODE REVIEW

**Comment by hjmjohnson** [comment]
@jcfr Are there any concerns with this cosmetic PR?

File: tclap/include/tclap/StdOutput.h [hunk 1]
```diff
@@ -164,30 +164,35 @@ inline void StdOutput::_shortUsage( CmdLineInterface& _cmd,
   XorHandler xorHandler = _cmd.getXorHandler();
   std::vector< std::vector<Arg*> > xorList = xorHandler.getXorList();
 
-  std::string s = progName + " ";
+  std::string s = progName + "";
```
Comment (lassoan) [inline tclap/include/tclap/StdOutput.h]: Was removal of the space between the quotation marks intentional? If yes, shouldn't we remove the whole literal (just keep `std::string s=progName;`)?

**Comment by lassoan** [comment]
I've added a minor comment but otherwise this looks good to me.

**Comment by jcfr** [comment]
Thanks for the patch. Before integrating, I suggest we:
* update verision of tclap using in SlicerExecutionModel (see #115). 
* add your patch into a branch publish at https://github.com/Slicer/tclap

**Comment by hjmjohnson** [comment]
@jcfr This seems like a lot of work. What is the benefit?

File: tclap/include/tclap/StdOutput.h (hunk 1 above)
Comment (hjmjohnson) [reply tclap/include/tclap/StdOutput.h]: @lassoan I was trying to minimize the differences from the original tclap code and sttill improve the resulting formatting style.  I will remove in the next revision.

**Comment by hjmjohnson** [comment]
@jcfr This is superseded by #120 that includes the updated version as mentioned in #115.
